### ISSUE TITLE
New dependency to be added to the 2.1.0 release

### ISSUE THREAD

**Issue by xiki-tempula** [alchemistry/alchemlyb#321]
List of new packages to be add to the conda meta.yml file
In #317 pyarrow is added as a dependency.
In #303 loguru is added as a dependency.

I plan to release 2.1.0 when #322 and #303 is merged.
Once #303 is merged, I will submit another PR to fix #295 .

**Comment by xiki-tempula** [comment]
@orbeckst I think this release is ready. I will wait for https://github.com/conda-forge/alchemlyb-feedstock/pull/10 to be merged and then I shall do a release.

**Comment by orbeckst** [comment]
Don't wait for conda-forge — create the 2.1.0 on alchemlyb first and get it on pypi.

Then we wait for auto-regro-cf bot to make a PR for the new release. We then modify that PR for the 2.1.0 release. Essentially, cf always comes after GitHub release/pypi.

**Comment by orbeckst** [comment]
Just follow https://github.com/alchemistry/alchemlyb/wiki/Developer-Guide#release-management

**Comment by xiki-tempula** [comment]
Thanks. I will create the pypi release now. I fear that auto-regro-cf bot will create a conda release that don't have the correct dependency and makes the user unhappy.

**Comment by orbeckst** [comment]
The PR from the bot should run into problems and not pass the tests because of missing deps.

**Comment by xiki-tempula** [comment]
I see. This is reassuring to hear.

**Comment by orbeckst** [comment]
2.1.0 is on conda-forge https://anaconda.org/conda-forge/alchemlyb/files
